src/mono/mono/metadata/mono-config.c

```diff
@@ -184,774 +184,6 @@ mono_get_machine_config (void)
-				if (p != NULL){
-					char *libdir = mono_native_getrootdir ();
-					size_t libdir_len = strlen (libdir);
-					char *result;
-					
-					result = (char *)g_malloc (libdir_len-strlen("$mono_libdir")+strlen(attribute_values[i])+1);
-					strncpy (result, attribute_values[i], p-attribute_values[i]);
-					strcpy (result+(p-attribute_values[i]), libdir);
-					g_free (libdir);
-					strcat (result, p+strlen("$mono_libdir"));
-					info->target = result;
-				} else 
-					info->target = g_strdup (attribute_values [i]);
-			} else if (strcmp (attribute_names [i], "os") == 0 && !arch_matches (CONFIG_OS, attribute_values [i]))
-				info->ignore = TRUE;
-			else if (strcmp (attribute_names [i], "cpu") == 0 && !arch_matches (CONFIG_CPU, attribute_values [i]))
-				info->ignore = TRUE;
-			else if (strcmp (attribute_names [i], "wordsize") == 0 && !arch_matches (CONFIG_WORDSIZE, attribute_values [i]))
-				info->ignore = TRUE;
-		}
-		if (!info->ignore)
-			mono_dllmap_insert_internal (info->assembly, info->dll, NULL, info->target, NULL);
-	} else if (strcmp (element_name, "dllentry") == 0) {
-		const char *name = NULL, *target = NULL, *dll = NULL;
-		int ignore = FALSE;
-		for (i = 0; attribute_names [i]; ++i) {
-			if (strcmp (attribute_names [i], "dll") == 0)
-				dll = attribute_values [i];
-			else if (strcmp (attribute_names [i], "target") == 0)
-				target = attribute_values [i];
-			else if (strcmp (attribute_names [i], "name") == 0)
-				name = attribute_values [i];
-			else if (strcmp (attribute_names [i], "os") == 0 && !arch_matches (CONFIG_OS, attribute_values [i]))
-				ignore = TRUE;
-			else if (strcmp (attribute_names [i], "cpu") == 0 && !arch_matches (CONFIG_CPU, attribute_values [i]))
-				ignore = TRUE;
-			else if (strcmp (attribute_names [i], "wordsize") == 0 && !arch_matches (CONFIG_WORDSIZE, attribute_values [i]))
-				ignore = TRUE;
-		}
-		if (!dll)
-			dll = info->dll;
-		if (!info->ignore && !ignore)
-			mono_dllmap_insert_internal (info->assembly, info->dll, name, dll, target);
-	}
-}
-
-static void
-dllmap_finish (gpointer user_data)
-{
-	DllInfo *info = (DllInfo *)user_data;
-
-	g_free (info->dll);
-	g_free (info->target);
-	g_free (info);
-}
-
-static const MonoParseHandler
-dllmap_handler = {
-	"dllmap",
-	dllmap_init,
-	dllmap_start,
-	NULL, /* text */
-	NULL, /* end */
-	dllmap_finish
-};
-#endif
-
-static void
-legacyUEP_start (gpointer user_data, 
-              const gchar         *element_name,
-              const gchar        **attribute_names,
-              const gchar        **attribute_values) {
-	if ((strcmp (element_name, "legacyUnhandledExceptionPolicy") == 0) &&
-			(attribute_names [0] != NULL) &&
-			(strcmp (attribute_names [0], "enabled") == 0)) {
-		if ((strcmp (attribute_values [0], "1") == 0) ||
-				(g_ascii_strcasecmp (attribute_values [0], "true") == 0)) {
-			mono_runtime_unhandled_exception_policy_set (MONO_UNHANDLED_POLICY_LEGACY);
```

Comment:
`mono_runtime_unhandled_exception_policy_set` is dead code now.  Could get rid of it too